web/modules/custom/collection/collection.behat.inc

```diff
@@ -58,6 +58,24 @@ class CollectionSubContext extends DrupalSubContextBase implements DrupalSubCont
   }
 
   /**
+   * Assert the order of elements.
+   *
+   * @Then I should see the following in the repeated :element element within the context of the :parentElement element:
```

Comment:
This is really unnecessary at this point. We only need this method for one specific case and it is really unfriendly for the user to include css tags in the behat step.
I propose to change this to @Then I should see the following collection menu items in the specified order:

If we are to need this function in the future for other cases, we can make a trait out of it or just move the code somewhere else. In my opinion it is better to hardcode the css tags in the code for now.